modules/ecs_service_fargate/outputs.tf

```diff
@@ -6,7 +6,11 @@ output "task_arn" {
   value = aws_ecs_task_definition.ecs_task.arn
 }
 
+# DEPRECATED: use task_role_id instead
```

Comment:
Because of the new versioning system, I'd recommend just removing this instead of marking it as deprecated. If someone needs the old behaviour, they can point to an earlier tag.

Reply:
That makes sense. Only ECO and CPI seems to be using it. CC: @Fitzsimmons @ecbrodie

Reply:
What is the "this" that you are referring to @ajrkerr @arturopie ?

Reply:
We renamed an output of the `ecs_service_fargate` module that CPI is using, so when you upgrade the module, you will have to change your code

Reply:
So, you were referring to the rename of `iam_id` to `task_role_id`?

Also, CC @akasztennyNulogy

Reply:
yes